### PR TITLE
Type Hints for AWs module: KMS, Lambda, Organizations

### PR DESCRIPTION
**Pull request by FAYiEKcbD0XFqF2QK2E4viAHg8rMm2VbjYKdjTg**
Also fixed a type that was wrong- regions should be list of str!

### CODE REVIEW

File: cartography/intel/aws/iam.py [hunk 1]
```diff
@@ -710,7 +710,7 @@ def sync_user_access_keys(
 
 @timeit
 def sync(
-    neo4j_session: neo4j.Session, boto3_session: boto3.session.Session, regions: str, account_id: str,
+    neo4j_session: neo4j.Session, boto3_session: boto3.session.Session, regions: List[str], account_id: str,
```
Comment (achantavy) [inline cartography/intel/aws/iam.py:713]: Can we standardize on `current_aws_account_id`?

File: cartography/intel/aws/kms.py [hunk 2]
```diff
@@ -321,7 +341,10 @@ def sync_kms_keys(neo4j_session, boto3_session, region, current_aws_account_id,
 
 
 @timeit
-def sync(neo4j_session, boto3_session, regions, current_aws_account_id, aws_update_tag, common_job_parameters):
+def sync(
+    neo4j_session: neo4j.Session, boto3_session: boto3.session.Session, regions: List[str], current_aws_account_id: str,
+    aws_update_tag: int, common_job_parameters: Dict,
```
Comment (achantavy) [inline cartography/intel/aws/kms.py:346]: Can we standardize on `update_tag`?

**Review by achantavy — CHANGES_REQUESTED** [review]

File: cartography/intel/aws/iam.py (hunk 1 above)
Comment (FAYiEKcbD0XFqF2QK2E4viAHg8rMm2VbjYKdjTg) [reply cartography/intel/aws/iam.py:713]: Yeah, I was gonna do that in a future PR where I homogenize variable and function names

File: cartography/intel/aws/kms.py (hunk 2 above)
Comment (FAYiEKcbD0XFqF2QK2E4viAHg8rMm2VbjYKdjTg) [reply cartography/intel/aws/kms.py:346]: Yeah, I was gonna do that in a future PR where I homogenize variable and function names

Comment (achantavy) [reply cartography/intel/aws/kms.py:346]: Roger

**Review by achantavy — APPROVED** [review]

**Pull request merged**